### ISSUE TITLE
RTL paragraph: 'Left to right' toolbar menu missing label and selected state semantics

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#42314]
### Description

When the admin language is a Right-To-Left language, the Paragraph block toolbar shows an additional dropdown menu to give users the ability to set the text direction to Left-To-Right:

<img width="884" alt="Screenshot 2022-07-11 at 10 04 44" src="https://user-images.githubusercontent.com/1682452/178221009-cf3f8398-9825-4c2f-9006-72dc0ead61b6.png">

- The toggle button is unlabelled: there's no text or aria-label to provide the button name.
- When activated, the 'Left to right' menu item is highlighted only visually. There's no semantics to indicate its selected state. This menu item should have a `role="menuitemcheckbox"` and an `aria-checked="false/true"` attribute.

### Step-by-step reproduction instructions

- Set the WordPress to a RTL language.
- Edit a post and add a Paragraph block.
- Select the block to make its toolbar appear.
- Inspect the toggle button that is placed on the left of the 'Align' button and observe it's unlabelled.
- Click the button to open the dropdown menu.
- Inspect the 'Left to Right' menu item button: observe there's no aria-checked=false attribute.
- Click 'Left to Right': the Paragraph text is now set to LTR and the menu closes.
- Open the menu again.
- Inspect the 'Left to Right' menu item button: observe there's no aria-checked=true attribute.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by afercia** [comment]
Fixing the `ParagraphRTLControl` would be pretty simple:
- Pass a `label` prop with meaningful text to `ToolbarDropdownMenu`.
- Pass `role: 'menuitemcheckbox'` to the only control in the `controls` prop array.

However, I'm not sure a dropdown menu with only one menu item is the best option in the first place. Users are forced to click twice to activate this settings, while a simple button would require just one click.

**Comment by afercia** [comment]
Looks like this control wrapper used to be a `ToolbarGroup` and changed to `DropdownMenu` in https://github.com/WordPress/gutenberg/pull/29863. Later, it changed to a `ToolbarDropdownMenu`. That may explain the lack of a label and semantic state, as the `ToolbarGroup` used to render a toggle button with a label and an aria-pressed state:

<img width="752" alt="Screenshot 2022-07-11 at 11 46 07" src="https://user-images.githubusercontent.com/1682452/178237383-24235ce6-6ac5-466d-a45f-240cca5505eb.png">
